ui/src/main.rs

```diff
@@ -252,8 +258,81 @@ fn app(cx: Scope) -> Element {
     let user_lang_saved = state.read().settings.language.clone();
     utils::language::change_language(user_lang_saved);
 
+    let pre_release_text = get_local_text("uplink.pre-release");
+
+    let theme = match &state.read().ui.theme {
```

Comment:
This could probably be simplified by doing

```suggestion
    let theme = state.read().ui.theme.as_ref().map(|theme| theme.styles.clone()).unwrap_or_default();
```